storage/src/adb/any/fixed/mod.rs

```diff
@@ -538,7 +538,7 @@ impl<
         max_ops: NonZeroU64,
     ) -> Result<(Proof<H::Digest>, Vec<Operation<K, V>>), Error> {
         let op_count = Location::new(op_count);
-        let end_loc = std::cmp::min(op_count, start_loc.checked_add(max_ops.get()).unwrap());
+        let end_loc = std::cmp::min(op_count, start_loc.saturating_add(max_ops.get()));
```

Comment:
Hmm, if we want to avoid panic here, seems better to return error?

(Though IMO panic! seems fine here, if you're overflowing here, something's pretty wrong.)